infra/README.md

```diff
@@ -0,0 +1,67 @@
+# Setting up your AWS Backend for TestGround
+
+In this directory, you will find:
+
+```
+» tree -L 1
+.
+├── README.md
+├── packer      # The script used to build the base Image we will run on an AWS EC2 Machine
+├── terraform   # The script used to spin up multiple EC2 instances
+└── ansible     # The scripts used to configure each of the instances
+```
+
+## Building a custom Image with Packer
+
+The packer directory contains a Hashicorp Packer template used to build the base AMI disk image for AWS. See [packer.io](https://www.packer.io/) for information on how to use Packer.
+
+**Note: If you are using the Protocol Labs Test Ground Infra AWS account, there will be an AMI image already published. You won't have to create one yourself.
+
+## Instantiating the backend with Terraform
+
+The terraform directory contains a Hashicorp Terraform configuration that can be used to provision resources needed to run a cluster on AWS - the EC2 virtual machines, an autoscaling group for the workers, plus a VPC (Virtual Private Cloud) for the network.
+
+The default cluster sets up a "manager" machine, another machine to run redis, and 2 worker machines for running tests on.
+
+Steps:
+
+- 1. Install the Terraform CLI [Terraform](https://www.terraform.io/).
+- 2. Get some AWS credentitals (Access Key ID and Secret Access Key). If you don't have a key pair, you can create one in the EC2 dashboard in the AWS web console.
+- 3. `cd terraform` and run `terraform init` to install all the deps necessary
+- 4. Create a file with the name `terraform.tfvars`. It should look like:
+
+```
+key_name = <ssh key pair name registered in AWS>
+tag      = <name for your cluster, use only alphanumeric chars and underscores>
+```
+
+The tag is used to name your cluster. It must be unique. Be careful not to re-use a tag that is already in-use, or your cluster might get joined into another one.
+
+- 5. To set up the resources on AWS, simple run `terraform apply`. Terraform will ask for you to type in `yes` as a confirmation step. The final output from Terraform will contain the public DNS name you can ssh to get into the manager node.
```

Comment:
When running this step, I get the following error:

```
» terraform apply

Error: No valid credential sources found for AWS Provider.
        Please see https://terraform.io/docs/providers/aws/index.html for more information on
        providing credentials for the AWS Provider

  on testground.tf line 2, in provider "aws":
   2: provider "aws" {
```

https://www.terraform.io/docs/providers/aws/index.html suggests storing the keys differently from what is described in this tutorial. 

I took my access key out of https://console.aws.amazon.com/iam/home?region=us-west-2#/security_credentials, although I note that it is not the name of an ssh key pair, just an access key ID.

@jimpick could you point me to the right page?

Reply:
AWS credential management is quite a complex topic. Fortunately, there is documentation here:

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html

I'm pretty sure if you have `~/.aws/credentials` configured, then Terraform will use it.

Reply:
That worked! Thank you!